extensions/amp-form/0.1/amp-form.js

```diff
@@ -561,6 +564,21 @@ export class AmpForm {
   }
 
   /**
+   * @return {Array<!Element>}
+   * @public
+   */
+  getDynamicElements() {
+    const dynamicElements = [];
+    const successDiv =
+        this.form_./*OK*/querySelector(`[${FormState_.SUBMIT_SUCCESS}]`);
+    const errorDiv =
+        this.form_./*OK*/querySelector(`[${FormState_.SUBMIT_ERROR}]`);
+    successDiv && dynamicElements.push(successDiv);
+    errorDiv && dynamicElements.push(errorDiv);
```

Comment:
IMO it's more readable to use conditionals when your statements have side effects. `||` or the ternary operator are typically used for returning values without side effects.